src/admin/content/content.const.ts

```diff
@@ -0,0 +1,17 @@
+import { ROUTE_PARTS } from '../../shared/constants';
+
+export const CONTENT_PATH = {
+	CONTENT: `/${ROUTE_PARTS.admin}/${ROUTE_PARTS.content}`,
+};
+
+export const CONTENT_OVERVIEW_TABLE_COLS = [
```

Comment:
add type:
```typescript
ContentOverviewTableCols: {id: ContentOverviewTableCols, label: string}[] = [ ...
```

Reply:
Preferrably met predefined interface ipv in-line interface.

Reply:
Fixed